manual/create-new-environment.md

```diff
@@ -0,0 +1,58 @@
+# Create a new environment
+This is a guide to creating a new tdr environment in a clean AWS account.
+
+## Create parameters in parameter store.
+There are five parameters which need to be set as they can't be created or found by terraform
+* /mgmt/aws_elb_account - The parent account where the logs are sent. Maintained by digital services so no way to have this in terraform
+* /mgmt/cost_centre - This is a TNA property which is not available to terraform
+* /mgmt/management_account - There's no way to get this while running terraform for other accounts
+* /mgmt/trusted_ips - The IP list for the WAF. We could probably reconfigure this to come from `tdr-configurations` but for now, this needs to be set.
+* /prod/frontend/auth/thumbprint - The thumbprint used for the IAM external provider. 
```

Comment:
Should `prod` be whatever the environment is? How do you work out what the value of this should be?

Reply:
Oh sorry, just read the next section. 🙂

Reply:
You're right about /prod though, I've changed it to /environment